packages/core/src/browser/theming.ts

```diff
@@ -201,7 +201,7 @@ export class BuiltinThemeProvider {
         id: 'dark',
         label: 'Dark Theme',
         description: 'Bright fonts on dark backgrounds.',
-        editorTheme: 'vs-dark',
+        editorTheme: 'dark-plus',
```

Comment:
this contradicts the values from `monaco.d.ts`, no? is this out-of-sync?

Reply:
I've put a comment to make it clearer where this is contributed.
The initial idea with the complicated overriding was that we would go with default monaco theme and then later reset it to the plus one. But as both monaco and the loading of the plus themes are coming from the monaco extension there is no point in doing that.